base_delivery_carrier_extensions/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# © 2020 FactorLibre - Zahra Velasco <zahra.velasco@factorlibre.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'Base delivery carrier extension',
+    'version': '11.0.1.0.0',
+    'depends': [
+        'delivery',
+        'base_delivery_carrier_label'
+    ],
+    'category': 'Stock',
+    'author': 'FactorLibre',
+    'license': 'AGPL-3',
+    'website': 'http://www.factorlibre.com',
```

Comment:
It should be the OCA GitHub repository one (you can put FL website in CONTRIBUTORS)

```suggestion
    'website': 'http://github.com/OCA/delivery-carrier',
```